### DISCUSSION TITLE
How to use .control command ?

### DISCUSSION THREAD

**Discussion by jojo2massol** [Adikso/BashBot#36] [Q&A]
I don't understant how $.control works. I tried .control ⏬ [DOWN] but it doesn't works. Emoji is under the terminal, but no reaction.

**Comment by Adikso** [comment]
It's "controls" not "control" and you missed "add".
`$.controls add ⏬ [DOWN]`
If you typed command correctly then make sure that the application used in terminal is really handling arrow down. What is this application?
